homeassistant/components/media_player/hdmi_cec.py

```diff
@@ -0,0 +1,64 @@
+"""
+Support for Vera switches.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.vera/
+"""
+import logging
+
+from homeassistant.components.hdmi_cec import CecDevice, CEC_DEVICES, CEC_CLIENT
+from homeassistant.components.media_player import MediaPlayerDevice
+
+DEPENDENCIES = ['hdmi_cec']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Find and return Vera switches."""
+    _LOGGER.info("setting CEC media players %s", CEC_DEVICES['media_player'])
+    add_devices(
+        CecMediaPlayer(hass, CEC_CLIENT, device) for
+        device in CEC_DEVICES['media_player'])
+
+
+class CecMediaPlayer(CecDevice, MediaPlayerDevice):
```

Comment:
This was just preparation for implementing volume up, volume down and other stuf. But removing from master until finished.